.travis.yml

```diff
@@ -8,7 +8,10 @@ stages:
 jobs:
   include:
     - stage: test
-      script: travis_wait 14 ./gradlew build
+      script: travis_wait 21 ./gradlew build
+    - script: travis_wait 21 ./gradlew -Djira.version=7.13.0 build
+    - script: travis_wait 21 ./gradlew -Djira.version=7.3.0 build
+    - script: travis_wait 21 ./gradlew -Djira.version=7.2.0 build
```

Comment:
Are those run in parallel?

Reply:
Yes, but some jobs are queued and start later due to lack of agents (AFAIK there's a limited OS agents pool).

Reply:
I've switched back to the matrix, as it creates more readable jobs. I still don't know, why it runs 8.0.0 twice.

Reply:
Thank god we can cover 4 versions of Jira here and we **do not** have to do it for all other tests in all the other modules.